proposals/lip-0048.md

```diff
@@ -25,15 +25,34 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
+The fee structure presented here serves to allow defining sufficient cost to manage state growth and prevent transaction pool spamming. In principle, any transaction resulting in a state change should incur a fee. This reasoning is implemented in the `payFee` method, which can be invoked by other modules.
+
 ### Fee Token ID
 
-Each chain can configure the token used to pay fees. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+Each chain can configure the token used to pay fees, as defined by `TOKEN_ID_FEE`. On the Lisk mainchain, the token used for transaction fees is the LSK token.
 
 ### Minimum Fee per Transaction
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when the minimum fee per byte is set to zero.
+As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee, which can be set to zero. For a transaction `trs`, the minimum fee is determined by its size and the proportionality constant `MIN_FEE_PER_BYTE`: 
+
+```python
+trsSize = length(encodeTransaction(trs))
+minFee = MIN_FEE_PER_BYTE * trsSize
+```
+
+Note that chains can freely configure their `MIN_FEE_PER_BYTE`, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when it is set to zero.
```

Comment:
```suggestion
Note that chains can freely configure their `MIN_FEE_PER_BYTE`, which can be even set to zero. The value `MIN_FEE_PER_BYTE` can not be modified, unless a hard fork occurs. For this reason, we allow sidechains the possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block for which the minimum fee does not apply. This could be useful, for example, for sidechains that want to use LSK as the fee token: initially, no LSK tokens exist in the sidechain, so the chain would not be able to have any transactions, unless `MIN_FEE_PER_BYTE = 0`. By allowing an initial exception period, users have some time window to transfer LSK tokens to this chain; after this time, `MIN_FEE_PER_BYTE` could apply with a reasonable non-zero value. 
```

Reply:
This is just a suggested text and perhaps can be improved; I hope you get the main point of the suggestion.

Reply:
Very good, thanks for the suggestion!